bin/runnable-upload.js

```diff
@@ -6,13 +6,16 @@ var runnable = require('../lib/runnable')
 
 program
   .arguments('<file>')
+  .arguments('<file> [dest]')
```

Comment:
this is what these two lines are generating:

```
  Usage: runnable-upload [options] <file> <file> [dest]
```

Remove the `.arguments` line above this one - you only need one...